data/json/items/gun/40x46mm.json

```diff
@@ -99,7 +99,7 @@
     "copy-from": "launcher_base",
     "looks_like": "m79",
     "type": "GUN",
-    "name": { "str_sp": "Milkor MGL" },
+    "name": { "str": "Milkor MGL" },
```

Comment:
I think it would be wise to also append grenade launcher to the end of this weapon’s name. If a non-gun person would not be able to tell what an AR-15 is, MGL would probably fly over their head. My scientific source? I showed the name to my decidedly non-gun-person girl friend and observed the subject's blank expression.